app/views/shared/_statements_table.html.haml

```diff
@@ -22,6 +22,9 @@
             %br
             - path = current_facility ? statement_path(s) : account_statement_path(s.account, s, format: :pdf)
             = link_to t("statements.pdf.download"), path
+            - if SettingsHelper.feature_on?(:send_statement_emails) && current_facility
+              %br
+              = link_to(t("statements.resend"), resend_emails_facility_statement_path(current_facility, s))
```

Comment:
Can we make this a `put` or ` post` (because this link has a side-effect).

Reply:
@rdunlop I couldn't find any useful guidance for an action like this that sends an email.  We're not altering records, which is why I went with `get`.  Do you prefer `put` or `post` here?